src/dbnode/persist/fs/retriever.go

```diff
@@ -126,18 +127,19 @@ func NewBlockRetriever(
 	scope := fsOpts.InstrumentOptions().MetricsScope().SubScope("retriever")
 
 	return &blockRetriever{
-		opts:            opts,
-		fsOpts:          fsOpts,
-		logger:          fsOpts.InstrumentOptions().Logger(),
-		queryLimits:     opts.QueryLimits(),
-		bytesReadLimit:  opts.QueryLimits().BytesReadLimit(),
-		seriesReadCount: scope.Counter("series-read"),
-		newSeekerMgrFn:  NewSeekerManager,
-		reqPool:         opts.RetrieveRequestPool(),
-		bytesPool:       opts.BytesPool(),
-		idPool:          opts.IdentifierPool(),
-		status:          blockRetrieverNotOpen,
-		notifyFetch:     make(chan struct{}, 1),
+		opts:                    opts,
+		fsOpts:                  fsOpts,
+		logger:                  fsOpts.InstrumentOptions().Logger(),
+		queryLimits:             opts.QueryLimits(),
+		bytesReadLimit:          opts.QueryLimits().BytesReadLimit(),
+		seriesReadCount:         scope.Counter("series-read"),
```

Comment:
while you're here, you could kill `seriesReadCount`. it's redundant with the `total` count metric in the limit